src/server.cc

```diff
@@ -99,6 +99,7 @@ bool Server::getServerInfo(const char* file_name, ServerInfo* info) {
       PathInfo pathInfo;
       pathInfo.handler_id = config.statements_[i]->tokens_[2];
       pathInfo.config = config.statements_[i]->child_block_.get();
+      pathInfo.handler = RequestHandler::CreateByName(pathInfo.handler_id.c_str());
```

Comment:
call the handler's `Init` function after using `RequestHandler::CreateByName`. You'll need to pull the handler's URI prefix from the config to pass to `Init`, along with the config itself